packages/react-scripts/scripts/start.js

```diff
@@ -11,6 +11,11 @@
 
 process.env.NODE_ENV = 'development';
 
+// Load environment variables from .env file. Surpress warnings using silent
+// if this file is missing. dotenv will never modify any environment variables
+// that have already been set.
+require('dotenv').config({silent: true});
```

Comment:
Can this be moved to `packages/react-scripts/config/env.js`? That's the only place where these environment variables would be used anyway and then we don't need this in each script separately?